CMakeLists.txt

```diff
@@ -87,6 +87,10 @@ if (HAVE_SYS_STAT_H)
     if (HAVE_FTELLI64)
         list(APPEND fstat -DHAVE_FTELLI64)
     endif ()
+    check_function_exists(_fseeki64 HAVE_FSEEKI64)
+    if (HAVE_FSEEKI64)
+        list(APPEND fstat -DHAVE_FSEEKI64)
```

Comment:
Do you know why the Windows (?) methods (`ftelli64` etc.) ending on `i64` are added to the `fstat` list, but the POSIX methods ending on `o` (`ftello` etc.) are getting their own individual lists?